<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroSessao</class>
 <widget class="QDialog" name="cadastroSessao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Agendar Sessão&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pesquisar paciente:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pesquisarPacAg"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tw_paciente_ses"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dataAgenda"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Horário:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="horaAgenda"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btnCadastrarSes">
         <property name="text">
          <string>Agendar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancelarSes">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
